x-pack/agentbeat/main.go

```diff
@@ -70,12 +70,8 @@ func prepareCommand(rootCmd *cmd.BeatsRootCmd) *cobra.Command {
 		// filename, as all the beats set this in the initialization.
 		err := cfgfile.ChangeDefaultCfgfileFlag(rootCmd.Use)
 		if err != nil {
-			panic(fmt.Errorf("failed to set default config file path: %w", err))
+			panic(fmt.Errorf("failed to set default config file path: %v", err))
```

Comment:
Should this be a %w instead?

Reply:
Even though I believe nothing will ever try to recover from this panic, I believe it's good to make the linter happy.

Reply:
Fixed.